tests/e2e/enform.spec.js

```diff
@@ -0,0 +1,298 @@
+      }
+      if (formFieldsAttributes[field.name]?.label?.length > 0 && field.type !== 'OPT') {
+        await page.locator(`tr[data-en-id="${field.id}"] input[type="text"]`).fill(formFieldsAttributes[field.name].label);
+      }
+      if (formFieldsAttributes[field.name]?.default_value?.length > 0) {
+        await page.locator(`tr[data-en-id="${field.id}"] .dashicons-edit`).click();
+        const defaultValField = await page.waitForSelector(`.dialog-${field.id} input[data-attribute="default_value"]`)
+        await defaultValField.fill(formFieldsAttributes[field.name].default_value)
+        await page.locator(`.dialog-${field.id} button[title='Close']`).click();
+      }
+    }
+
+    // Set dependency field value.
+    for (const field of fields) {
+      if (!formFieldsAttributes[field.name]?.dependency) {
+        continue;
+      }
+      await page.locator(`#en_form_selected_fields_table tr[data-en-id="${field.id}"] .dashicons-edit`).click();
+      const element = await page.waitForSelector(`.dialog-${field.id} .dependency-select`);
+      await element.selectOption(formFieldsAttributes[field.name].dependency);
+      await page.locator(`.dialog-${field.id} button[title='Close']`).click();
+    }
+
+    // Reorder last field with the second to last field to assert that ordering fields works.
+    const source = page.locator('tr:last-child span.dashicons-sort');
+    const target = page.locator('tr:nth-last-child(2) span.dashicons-sort');
+    await source.dragTo(target);
+
+    await page.locator('#publish').click();
+
+    // Assert that the fields were saved with the expected attributes/values.
+    for (const field of fields) {
+      const element = await page.locator(`tr[data-en-id="${field.id}"]`);
+      expect(element).toHaveAttribute('data-en-name', field.name);
+      if (formFieldsAttributes[field.name]?.required) {
+        await page.locator(`tr[data-en-id="${field.id}"] input[type="checkbox"]`).isChecked();
+      }
+      if (formFieldsAttributes[field.name]?.label?.length > 0 && field.type !== 'OPT') {
+        const labelField = await page.locator(`tr[data-en-id="${field.id}"] input[type="text"]`);
+        expect(labelField).toHaveValue(formFieldsAttributes[field.name].label);
+      }
+      if (formFieldsAttributes[field.name]?.default_value?.length > 0) {
+        await page.locator(`tr[data-en-id="${field.id}"] .dashicons-edit`).click();
+        const defaultValField = await page.locator(`.dialog-${field.id} input[data-attribute="default_value"]`)
+        await expect(defaultValField).toHaveValue(formFieldsAttributes[field.name].default_value);
+        await page.locator(`.dialog-${field.id} button[title="Close"]`).click();
+      }
+    }
+
+    const currentUrl = await page.url();
+    enFormId = new URL(currentUrl).searchParams.get('post');
+    console.log(`form ID: ${enFormId}`);
+  });
+
+  test('create Page with EN Form side style', async ({page, context}) => {
+    test.setTimeout(120 * 1000);
```

Comment:
[Timeout has been changed](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/main/playwright.config.js#LL10C3-L10C3) so maybe this is no longer relevant?